docusaurus.config.js

```diff
@@ -40,7 +40,7 @@ const redocusaurus = [
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: "ConsenSys Web3Signer",
+  title: "Consensys Web3Signer",
```

Comment:
Not sure which way is correct but maybe this should be Web3signer too

Reply:
Maybe we should do Web3Signer -> Web3signer in a separate PR? We'll need to ensure the codebase gets updated as well, see: https://github.com/Consensys/web3signer/blob/master/CONTRIBUTING.md

Reply:
I don't know which is better/more correct. If there's a requirement to change it though then agree separate PR is best

Reply:
Will confirm which is the preferred name, and update it (if required) in a seperate PR.

Reply:
@bgravenorst just submitted more comments